lib/transforms/base-scss.js

```diff
@@ -24,7 +24,10 @@ module.exports = function (options, output) {
 		]);
 		output.sassLoader = {
 			sourcemap: true,
-			includePaths: [ path.resolve('./bower_components') ],
+			includePaths: [
+				path.resolve('./bower_components'),
+				path.resolve('./node_modules')
```

Comment:
A good question.
In discussion with @i-like-robots we moved to doing it this way so that it became clear in the code that you were using it from node_modules rather than bower.

I don't think that there should really be any performance issue in having it at this less specific level.